UI/data/themes/Dark.qss

```diff
@@ -60,6 +60,7 @@ OBSTheme {
 }
 
 OBSTheme::disabled {
+    windowText: rgb(153,153,153);
```

Comment:
Can we bump this up to rgb(171,171,171) for better contrast?
https://webaim.org/resources/contrastchecker/?fcolor=ABABAB&bcolor=3A393A

This may also require us to bump up the default windowText color from rgb(225,224,225) to distinguish better

Reply:
maybe done.